### ISSUE TITLE
 Investigatory work on Free Tier for App Center Data

### ISSUE THREAD

**Issue by Zakeelm** [microsoft/appcenter#799]
One of the newest items we want to bring to the suite of MBaaS services is a free tier. We realize the importance of getting started with no cost and want to give developers the ability to do so. We're actively investigating the implementation of our free tier and working out the best way to accomplish this. As of now, our plan for App Center Data is to have a free tier that supports 400 Request Units (RUs) per second, which is the minimum provisioned throughput in Cosmos DB with no major limits on App Center Data functionality. The free tier will enable you to seamlessly get started without an Azure subscription. Developers will be able to use both the client SDKs and the portal. We'll be spending the month of August speaking with developers to learn about your needs for the Data Free tier, so that we can create the best getting started experience. 
To that end, we'll spend the month of Aug speaking with developers to learn about your needs for a free tier for Auth.

We're actively looking for developers to speak with and help us shape our product. Please leave a comment below if you'd like to collaborate together!

**Comment by ttirrell** [comment]
I'm very interested in the Free Tier.  I currently am using Easy Tables and want to switch to Cosmos DB, but I am unsure if I should start now or wait for the Free Tier.  How long till this is available?  Thanks.

**Comment by Zakeelm** [comment]
Hi there! 

Apologies for the delayed response. We're estimating that the free tier will be done at some point a bit later this year!

**Comment by Zakeelm** [comment]
Update on this: We're still ironing out some of the kink in our free tier. We're still currently heads down on the investigatory work. Should have more details on an ETA for this work item on a later date.

**Comment by RyanGaudion** [comment]
Any Update? If not I'll have to use AWS

**Comment by RyanGaudion** [comment]
Thanks for the link but this says "Continue Free Tier investigatory work". Can we please have a rough estimate as to when we can expect to see this to progress from investigatory work into implemnentation?

**Comment by Zakeelm** [comment]
@curelom  We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue as it's no longer planned work.
